### ISSUE TITLE
Unable to install the report

### ISSUE THREAD

**Issue by ripunjayasaran** [TitouanVanBelle/XCTestHTMLReport#73]
Using `bash <(curl -s https://raw.githubusercontent.com/TitouanVanBelle/XCTestHTMLReport/master/install.sh)` to download the latest version of the tool does not work anymore on the CI builds. The error that i get is:

> Downloading xchtmlreport from master

> Failed to download XCTestHTMLReport. Make sure the version you're trying to download exists.

Also, using the homebrew formula `brew install TitouanVanBelle/XCTestHTMLReport/xchtmlreport` throws an error as well:

> fatal: ambiguous argument 'refs/remotes/origin/master': unknown revision or path not in the working tree.
> Use '--' to separate paths from revisions, like this:
> 'git <command> [<revision>...] -- [<file>...]'
> Error: No available formula with the name "titouanvanbelle/xctesthtmlreport/xchtmlreport" 
> ==> Searching for a previously deleted formula (in the last month)...
> Error: No previously deleted formula found.
> ==> Searching for similarly named formulae...
> ==> Searching local taps...
> Error: No similarly named formulae found.

Note: using `bash <(curl -s https://raw.githubusercontent.com/TitouanVanBelle/XCTestHTMLReport/master/install.sh) '1.5.0'` works though.

It would be nice if this could be fixed soon as it breaks using this tool on CI builds

**Comment by ripunjayasaran** [comment]
cc: @TitouanVanBelle

**Comment by chrisballinger** [comment]
@ripunjayasaran This is resolved in #74  

Try:

```
bash <(curl -s https://raw.githubusercontent.com/TitouanVanBelle/XCTestHTMLReport/ae1281af59618f2719c6b3629bf6f139ee2257a3/install.sh)
```

**Comment by ripunjayasaran** [comment]
@chrisballinger : Thanks a lot for the timely fix. Any chance we could release this with 1.6.0 so that we could use `bash <(curl -s https://raw.githubusercontent.com/TitouanVanBelle/XCTestHTMLReport/master/install.sh)` instead?

**Comment by ripunjayasaran** [comment]
@chrisballinger : Thanks a lot. Much appreciated.

**Comment by chrisballinger** [comment]
Ok it’s merged. Did that resolve your CI issue?

**Comment by ripunjayasaran** [comment]
@chrisballinger : Going to try out shortly and let you know. Thanks again for all your work!

**Comment by TitouanVanBelle** [comment]
Thanks for the report and the fix! Didn't pay attention to that after releasing.

**Comment by ripunjayasaran** [comment]
@chrisballinger : The issue is fixed now. I can now get the latest 1.6.0 fixes via the regular command.

**Comment by ripunjayasaran** [comment]
@TitouanVanBelle : would it safe to assume that changes for 1.6.0 have not been released yet? Using `bash <(curl -s https://raw.githubusercontent.com/TitouanVanBelle/XCTestHTMLReport/master/install.sh)` to download the report, states the version of the tool at 1.6.0, but if i use `xchtmlreport -h` i see the version as 1.5.0. Also, i don't see the fix for #70 when i installed the report a few minutes back. Am i missing something here?

**Comment by ripunjayasaran** [comment]
I think all your changes & bug fixes for 1.6.0 are on the develop branch and have not been merged to master yet. However, changing the branch name to get the fixes using `bash <(curl -s https://raw.githubusercontent.com/TitouanVanBelle/XCTestHTMLReport/develop/install.sh)` doesn't work either. cc: @chrisballinger

**Comment by chrisballinger** [comment]
@ripunjayasaran The release is right here: https://github.com/TitouanVanBelle/XCTestHTMLReport/releases/tag/1.6.0

The binary attached to the release was compiled with the `1.6.0` git tag, which should include a fix for #70. https://github.com/TitouanVanBelle/XCTestHTMLReport/releases/download/1.6.0/xchtmlreport-1.6.0.zip

I'm guessing the output is printing the wrong version due to a bug in the code, because it is showing the new `-j` option which was not present in 1.5.0.

```bash
$ xchtmlreport -v
XCTestHTMLReport 1.5.0
Error: Argument -r is required

Usage: xchtmlreport -r <resultBundlePath>

Options:
    -h               Print usage and available options
    -v               Provide additional logs
    -j               Provide JUnit XML output
    -r               Path to a result bundle (allows multiple)
```
